db/migrate/20210218022053_solved_fix_high_auto_close_topic_hours.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+class SolvedFixHighAutoCloseTopicHours < ActiveRecord::Migration[6.0]
+  def up
+    execute <<-SQL
+      UPDATE site_settings
+         SET value = '175000'
```

Comment:
Should this be 175200?

Reply:
@tgxworld I wanted to do it just below the max so it’s not on the absolute limit :). They only get 10 days of auto close off of 20 years lol